foursight_core/boto_sqs.py

```diff
@@ -0,0 +1,25 @@
+# The primary/initial purpose of this was to be able to use the SQS_URL environment variable to refer
+# to a locally running ersatz version of SQS via an emulator like localstack (https://localstack.cloud).
+
+import boto3
+import os
+
+
+def boto_sqs_client(**kwargs):
```

Comment:
As to naming, I'm wondering if it shouldn't have boto3 (not just boto) in its name so that searches for boto3 in the code would turn it up. If you change the order of tokens, using boto3_client_sqs, then searches for 'boto3.client.*sqs' can find `boto3.client('sqs')` and `boto3.client("sqs")`. This may not matter, but I'm trained to think about grep kinds of things as part of naming to promote systems with good discovery. In the Lisp community there's a function "apropos" that one uses to find all the defined names with a given string in any module. It doesn't search sources, it searches the in-memory set of names for each package (vaguely like a module).

Alternatively, a name like `mockable_boto3_sqs_client` might give a sense of why the function was being called.

Reply:
This stuff is being superseded by the monkey patching based approach.